lib/matplotlib/tests/test_category.py

```diff
@@ -269,3 +270,23 @@ def test_mixed_type_update_exception(self, ax, plotter, xdata):
         with pytest.raises(TypeError):
             plotter(ax, [0, 3], [1, 3])
             plotter(ax, xdata, [1, 2])
+
+
+@pytest.mark.style('default')
+@check_figures_equal(extensions=["png"])
+def test_overliding_units(fig_test, fig_ref):
```

Comment:
"overliding"?

Reply:
l and r are next to each other on my keyboard.  I wish I could blame all of my miss-spellings on poor typing as easily as this one...

Reply:
ah, dvorak? :)